doc/python/webgl-vs-svg.md

```diff
@@ -58,6 +59,15 @@ In addition to the above limitations, the WebGL-powered version of certain SVG-p
 * Range breaks on time-series axes are not yet supported
 * Axis range heuristics may differ
 
+### Multiple WebGL Contexts
+
+*New in 5.19*
+
+If you encounter WebGL context limits when rendering many WebGL-based figures on one page, you can use [Virtual WebGL](https://github.com/greggman/virtual-webgl).
```

Comment:
```suggestion
Most browsers have a limit of between 8 and 16 WebGL contexts per page. Each Plotly WebGL-based figure uses one context, so if you exceed your browser's limit, not all figures will render and you'll see an error. In the console in Chrome, for example, you'll see the error: "Too many active WebGL contexts. Oldest context will be lost".

If you encounter WebGL context limits when rendering WebGL-based figures on one page, you can use [Virtual WebGL](https://github.com/greggman/virtual-webgl).
```

For me I found in testing that it was 8, but I've seen other mentions of people encountering a limit at 16. And 8-16 is mentioned here https://github.com/greggman/virtual-webgl